### DISCUSSION TITLE
Actions on release aren't running

### DISCUSSION THREAD

**Discussion by evert** [community/community#25709] [Actions]
<p>We had an action for an open source repo, that automatically creates and publishes npm packages ‘on release’. This worked until yesterday.</p>
<p>It’s a classic: “Nothing changed but it stopped working”-issue, which might mean we <em>did</em> change something but I don’t know what.</p>
<p>This has happened for our last 2 releases, 2 different repos:</p>
<p>Repo 1:</p>
<pre><code class="lang-auto">https://github.com/curveball/a12n-server
https://github.com/curveball/a12n-server/releases/tag/v0.19.12
https://github.com/curveball/a12n-server/blob/a3241299d77ec16c8cfc2b3e549dc6f8ddd2102e/.github/workflows/npm-publish.yml
Action list: https://github.com/curveball/a12n-server/actions/workflows/npm-publish.yml
</code></pre>
<p>Repo 2:</p>
<pre><code class="lang-auto">https://github.com/curveball/browser
https://github.com/curveball/browser/releases/tag/v0.16.4
https://github.com/curveball/browser/blob/dda822f630cd5bbce5789d71a8e2d3164697ea4f/.github/workflows/npm-publish.yml
Action list: https://github.com/curveball/browser/actions/workflows/npm-publish.yml
</code></pre>
<p>Any guidance with this is appreciated!</p>
<p>(I had to put the links in code blocks in order to avoid the ‘maximum 2 links’ filter)</p>

**Comment by irongut** [comment]
<p>I have a test repo for Actions, one of which is triggered on release. I’ve just tried it and my release action runs as expected. One difference I notice is you are triggering when a release is created but my actions trigger when a release is published. I don’t know if that will make a difference but it might be worth trying if your release actions still aren’t running.</p>
<pre><code class="lang-auto">on:
  release:
    types: [published]
    branches: [master]
</code></pre>

**Comment by evert** [comment, marked as answer]
<p>This must have been a temporary failure because this did resolve itself eventually!</p>
